database/README.md

```diff
@@ -245,13 +245,13 @@ FirebaseRecyclerAdapter adapter = new FirebaseRecyclerAdapter<Chat, ChatHolder>(
 
 The `FirebaseRecyclerPagingAdapter` binds a `Query` to a `RecyclerView` by loading documents in pages.
 This results in a time and memory efficient binding, however it gives up the real-time events
-afforded by the `FirestoreRecyclerAdapter`.
```

Comment:
I wonder how long we been living with this copy-paste error :grin:

Reply:
Haha oh no!